src/google/protobuf/compiler/objectivec/objectivec_helpers.cc

```diff
@@ -120,6 +125,13 @@ PrefixModeStorage::PrefixModeStorage() {
   if (exception_path) {
     exception_path_ = exception_path;
   }
+
+  // This one is a not expected to be common, so it doesn't get a generation
+  // option, just the env var.
+  const char* prefix = getenv("GPB_OBJC_USE_PACKAGE_AS_PREFIX_PREFIX");
```

Comment:
PREFIX_PREFIX?

Reply:
Yea, turning it back into words it make a little more sense: *GPB ObjC UsePackageAsPrefix Prefix*.  A few lines above it one using `GPB_OBJC_USE_PACKAGE_AS_PREFIX`, it conceptional is an addition to that: a prefix to use when that is enabled.